lib/server.js

```diff
@@ -1926,6 +1926,7 @@ WalletService.prototype.createTx = function(opts, cb) {
           customData: opts.customData,
           inputs: opts.inputs,
           fee: opts.inputs && !_.isNumber(opts.feePerKb) ? opts.fee : null,
+          noShuffleOutputs: opts.noShuffleOutputs
```

Comment:
Looks good. Could you please add a line for this param in the comment section of the method?